core-ui/src/shared/components/PersistentVolumesList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ComponentForList } from 'shared/getComponents';
+
+export function PersistentVolumesList(storageclass) {
+  const params = {
+    // Uncoment and check flow after resolve issue #537
+    // hasDetailsView: true,
+    fixedPath: true,
+    resourceUrl: `/api/v1/persistentvolumes`,
+    resourceType: 'persistentvolumes',
```

Comment:
```suggestion
    resourceType: 'PersistentVolumes',
```
We have a mechanism that splits the name by upper case letters to display it nicely in the list header. (Yes, I also agree that we should improve it).
Same for the `persistentvolumeclaims`